libs/velo-external-db-core/src/types.ts

```diff
@@ -23,8 +23,15 @@ export interface Payload {
     totalCount?: number;
 }
 
+export interface SchemaPayload {
```

Comment:
IMHO it should be `CollectionPayload`, because you used the word collection in the `RequestContect.operation`